### ISSUE TITLE
With Document.ToOrchidType deprecated, is there a new suggested workflow?

### ISSUE THREAD

**Issue by Chrono9084** [erfajo/OrchidForDynamo#165]
### Revit, Dynamo and Orchid version
Revit: 23.0.20.21
Dynamo: 2.13.1.5510
Orchid: 213.2.0.8293

### The issue
I have a graph where a user selected a titleblock within a project environment. The graph then feeds that document to the Document.ToOrchidType node and then proceeds to feed that into FamilyType.GetNested node.  The FamilyDocument Open node is from Genius Loci.

Now that the Document.ToOrchidType has been deprecated, I am not sure how to proceed.  Have you created a new workflow that you suggest?
![image](https://user-images.githubusercontent.com/37596510/192844007-957b4054-aa99-437e-8bf4-8efd4cd42d7d.png)
![image](https://user-images.githubusercontent.com/37596510/192844109-a8f50bfe-e331-4f17-901f-cdaf5aded359.png)

**Comment by erfajo** [comment]
There have been huge changes in this area... and I have written it in the log file for changes in the new versions like I have written it on LinkedIn and on the DynamoBIM forum... https://forum.dynamobim.com/t/the-fusion-post-for-coders

The new thing is, that I have made a "converter" that can bypass the protected classes by the dynamo team. Anyone can now "cast" the types into dynamo types even if they are shielded. By being able to do this I deprecated all my types if there exists a dynamo type... 

In other words, I don't have an Orchid Document type anymore, there is only the Dynamo Document type. There is no need to convert anymore for the user :-)

**Comment by Chrono9084** [comment]
Good to know!  Thank you for the quick response!

**Comment by Chrono9084** [comment]
Closing this.
